.github/workflows/integration.yml

```diff
@@ -0,0 +1,39 @@
+name: .NET Integration Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.x
+      - name: Restore Dependencies
+        run: cd src && dotnet restore
+      - name: Build
+        run: cd src && dotnet build --configuration Release --no-restore
+      - name: Set up Aerospike Database
+        uses: reugn/github-action-aerospike@v1.1.0
+      - name: Integration Tests
+        run: |
+          cd ./tests/AeroSharp.IntegrationTests/
+          dotnet test --no-restore -nodereuse:false \
+            /p:UseSharedCompilation=false \
+            /p:UseRazorBuildServer=false \
+            /p:CollectCoverage=true \
+            /p:CoverletOutputFormat=opencover \
+            /p:CoverletOutput=/home/runner/work/AeroSharp/AeroSharp/ \
```

Comment:
Is this full folder path required, or are you adding it just to be explicit?

Reply:
Tried with shortened path but it doesn't seem to work. Going to keep it as full path.